spec/support/jobseeker_helpers.rb

```diff
@@ -0,0 +1,22 @@
+module JobseekerHelpers
```

Comment:
I can see a page object here

Reply:
Haha me too. I will have a go in a different PR